docs/source/package_reference/utilities.md

```diff
@@ -50,6 +50,13 @@ These are basic dataclasses used throughout 🤗 Accelerate and they can be pass
 
 [[autodoc]] utils.ProjectConfiguration
 
+## Environmental Variables
+
+These are environmental variables that can be enabled for different use cases
+
+* `ACCELERATE_SEED` (`int`): The seed to use for different internal random states not covered through the base cases. Also set during [`utils.set_seed`]
```

Comment:
The description is a bit vague. As a user, what would interest me is in what circumstances I have to set this env var for reproducibility.

Reply:
Went with new implementation, irrelevent :)